manual/cops_rspec.md

```diff
@@ -1932,6 +1932,28 @@ expect(foo).to receive(:bar).at_most(:twice).times
 
 * [http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ReceiveCounts](http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ReceiveCounts)
 
+## RSpec/ReceiveNever
+
+Enabled by default | Supports autocorrection
+--- | ---
+Enabled | Yes
+
+Prefer `not_to receive(...)` over `receive(...).never`.
+
+### Examples
+
+```ruby
+# bad
+expect(foo).to receive(:bar).never
+
+# good
+expect(foo).not_to receive(:bar)
+```
+
+### References
+
+* [http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ReceiveNever](http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ReceiveNever)
```

Comment:
This was originally intended to be used for links to blog posts and whatever, complementary to a standard "Style Guide" link.

Reply:
The rake tasks intercepts "StyleGuide" from the config, as we have full url there :/